Source/Particles/ParticleIO.H

```diff
@@ -49,7 +50,7 @@ particlesConvertUnits (ConvertDirection convert_direction, T_ParticleContainer*
 #ifdef AMREX_USE_OMP
 #pragma omp parallel if (Gpu::notInLaunchRegion())
 #endif
-        for (WarpXParIter pti(*pc, lev); pti.isValid(); ++pti)
+        for (ParIter pti(*pc, lev); pti.isValid(); ++pti)
         {
             // - momenta are stored as a struct of array, in `attribs`
             auto& attribs = pti.GetAttribs();
```

Comment:
I looked at this some more. As is, this doesn't compile since the `ParIter` class doesn't have a `GetAttribs` method. This can be made to compile by replacing this line with the direct call instead of GetAttribs,
```
auto& attribs = pti.GetStructOfArrays().GetRealData();
```
However, I haven't checked if it runs and produces the correct result. Something for next week.